File: blimp/chunks/lang/benchmarks/fib.py
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

start = time.time()
result = fib(30)
elapsed = time.time() - start
print(f"{result} ({elapsed*1000:.1f}ms)")

File: blimp/chunks/lang/benchmarks/fib.rs
use std::time::Instant;

fn fib(n: i64) -> i64 {
    if n <= 1 { return n; }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let start = Instant::now();
    let result = fib(30);
    let elapsed = start.elapsed();
    println!("{} ({:.1}ms)", result, elapsed.as_secs_f64() * 1000.0);
}
